docs/content/en/docs/clustermgmt/cluster-upgrades/airgapped-upgrades.md

```diff
@@ -5,39 +5,26 @@ weight: 20
 aliases:
    /docs/tasks/cluster/cluster-upgrades/airgapped-upgrades/
 description: >
-  How to perform eks-anywhere upgrade for an airgapped cluster
+  Upgrading EKS Anywhere clusters in airgapped environments
 ---
-If you want to upgrade EKS Anywhere version, or your cluster upgrade requires EKS Anywhere version upgrade in airgapped environment, perform the following steps to prepare new artifacts in your registry mirror:
+The procedure to upgrade EKS Anywhere clusters in airgapped environments is similar to the procedure for creating new clusters in airgapped environments. The only difference is that you must upgrade your `eksctl anywhere` CLI before running the steps to download and import the EKS Anywhere dependencies to your local registry mirror.
 
-1. [Upgrade EKS Anywhere version]({{< relref "./vsphere-and-cloudstack-upgrades.md#eks-anywhere-version-upgrades" >}}).
+### Prerequisites
+- An existing [Admin machine]({{< relref "../../getting-started/install" >}})
+- **The upgraded version of the `eksctl anywhere` CLI installed on the Admin machine**
+- Docker running on the Admin machine
+- At least 80GB in storage space on the Admin machine to temporarily store the EKS Anywhere images locally before importing them to your local registry. Currently, when downloading images, EKS Anywhere pulls all dependencies for all infrastructure providers and supported Kubernetes versions.
+- The download and import images commands must be run on an amd64 machine to import amd64 images to the registry mirror.
```

Comment:
@d8660091 thoughts?

Reply:
When user run "download images" in their admin machine, the "docker pull" will run to pull the images. And since "docker pull" only pull images matching the admin machine's arch, arm64 admin machine will eventually upload only arm64 images to the registry mirror.

Reply:
So I guess I think the requirement is if you're running in an airgapped environment, you need to have an amd64 admin machine

Reply:
I added the following note to the admin machine page 
> - If you are running in an airgapped environment, the Admin machine must be amd64.